include/podio/SIOBlock.h

```diff
@@ -1,24 +1,32 @@
 #ifndef PODIO_SIOBLOCK_H
 #define PODIO_SIOBLOCK_H
 
+#include "podio/CollectionBuffers.h"
+
 #include <podio/CollectionBase.h>
 #include <podio/CollectionIDTable.h>
 #include <podio/GenericParameters.h>
 #include <podio/podioVersion.h>
 #include <podio/utilities/TypeHelpers.h>
-
 #include <sio/block.h>
+#include <sio/definitions.h>
 #include <sio/io_device.h>
 #include <sio/version.h>
 
+#include <cstddef>
+#include <cstdint>
+#include <functional>
 #include <map>
 #include <memory>
 #include <string>
 #include <string_view>
 #include <tuple>
+#include <unordered_map>
+#include <utility>
 #include <vector>
 
 namespace podio {
+class GenericParameters;
```

Comment:
This is already included above, so I don't think we need the forward declaration (or we can get rid of the include).

Reply:
The forward declaration is not there anymore.